src/libasr/pass/global_stmts.cpp

```diff
@@ -119,7 +120,7 @@ void pass_wrap_global_stmts(Allocator &al,
         /* a_body */ body.p,
         /* n_body */ body.size(),
         /* a_return_var */ (return_var ? return_var_ref : nullptr),
-        (return_var ? ASR::abiType::BindC : ASR::abiType::Source),
+        ASR::abiType::Source,
```

Comment:
I have changed this because `asr_to_llvm` changes the final function depending on ABI type (Look at https://github.com/lcompilers/lpython/blob/476d9bb47564f80158fe7e9f9b9124417fbed606/src/libasr/codegen/asr_to_llvm.cpp#L3794-L3805). We don't want the function name to change, it becomes difficult to figure out the correct function name in `PythonCompile.evaluate`.